bin.js

```diff
@@ -16,16 +20,22 @@ function print (...lines) {
 
 async function run () {
   let arg = process.argv[2] || ''
+  let filter = process.argv[3] || ''
```

Comment:
The filter can be passed before `--update`

Reply:
Are shure about give ability to set filter before args?
For example, all CLIs that working with paths accept it as last argument

Reply:
Yeap. User may call `npx print-snapshots server`, found problem, press arrow up to load latest command in terminal and add `--watch` in the end.

By the way, did we forget about changing `--help`?

Reply:
Ok, i will fix it and update help output. Thank you!